### PR TITLE
Bigclown

### PR DESCRIPTION
**Pull request by Cynerd**
Maintainer: me (review @commodo, @karlp ?)
Compile tested: Turris Omnia, Mox
Run tested: Turris Omnia

Description:
This is minimal set of packages to allow OpenWRT router to be used a gateway for Bigclown.

This is continuation of #8133 

Note: this PR requires a lot of new Python packages and those are pulled separately in PR #8378. This means that until those are merged and this rebase on top of them, the tests are going to be failing.

### CODE REVIEW

File: utils/bigclown/bigclown-gateway/Makefile [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+include $(INCLUDE_DIR)/package.mk
+include ../../../lang/python/python3-package.mk
+
+define Package/$(PKG_NAME)
+  SECTION:=utils
+  CATEGORY:=Utilities
+  SUBMENU:=BigClown
+  URL:=https://github.com/bigclownlabs/bch-gateway
+  TITLE:=BigCLown gateway
+  DEPENDS:=+kmod-usb-serial-ftdi +kmod-usb-acm +python3-click-log +python3-paho-mqtt +python3-pyserial +python3-yaml +python3-simplejson +python3-schema +python3-appdirs
```
Comment (commodo) [inline utils/bigclown/bigclown-gateway/Makefile]: it's nice to break this on multiple lines;

**Comment by commodo** [comment]
1 minor comment

[LGTM](https://static.boredpanda.com/blog/wp-content/uploads/2018/08/funny-coworkers-17-5b7e5fd7f3278__605.jpg)

File: utils/bigclown/bigclown-gateway/Makefile (hunk 1 above)
Comment (Cynerd) [reply utils/bigclown/bigclown-gateway/Makefile]: Broken..

**Comment by Cynerd** [comment]
And the last one. Also now rebased on master so in correct order. I would like to ask someone to merge it. @hnyman please please, thank you. :smiley:

**Pull request merged**